tools/unix/version.sh

```diff
@@ -42,8 +42,17 @@ function android_code {
   echo "${cutYear//./}$(printf %02d "$COUNT")"
 }
 
-function git_hash {
-  git describe --match="" --always --abbrev=8 --dirty
+function qt_int_version {
+  # yy_MM_dd
+  # yy - year
+  # MM - month
+  # dd - day
+  local cutYear=${DATE:2}
+  echo "${cutYear//./}"
```

Comment:
Why do you need a custom version for Qt? Let's reuse the same as for other platforms.

Reply:
Actually, Qt version doesn't really matter, but I tried to preserve format added before my patch.

Reply:
Int app version should be the same for all platform like 220816. It will be used in for dynamic data updates.

Reply:
Yes, of course, intVersion() is the same for all platforms - 220816. I don't think that we need one more section here (220816 instead of 22081601).